builders/gcp/base/acceptance/go_test.go

```diff
@@ -90,6 +90,14 @@ func TestAcceptanceGo(t *testing.T) {
 			FilesMustExist:    []string{"/layers/google.go.build/bin/main"},
 			FilesMustNotExist: []string{"/layers/google.go.runtime", "/workspace/main.go"},
 		},
+		{
+			Name:              "clear source with excludes",
```

Comment:
It looks like this test is failing, does it pass for you locally? (Same with the Java test, but that one may have been a flake.)

```
   TestAcceptanceGo/clear_source_with_excludes/cache_true: acceptance.go:216: Error running structure tests: exit status 1, logs:
        
        ====================================
        ====== Test file: config.yaml ======
        ====================================
        
        ====================================
        ====== Test file: config.json ======
        ====================================
        === RUN: Metadata Test
        --- PASS
        duration: 0s
        === RUN: File Existence Test: /layers/google.go.build/bin/main
        --- PASS
        duration: 0s
        === RUN: File Existence Test: /workspace/main.go
        --- FAIL
        duration: 0s
        Error: Error examining file in container: Error retrieving file from container: API error (404): Could not find the file /workspace/main.go in container e570001413b36b3df1d46ae7c919ba37b5224cd2e12b4260fd6aebe110d77744
        === RUN: File Existence Test: /layers/google.go.runtime
        --- PASS
        duration: 0s
        
        =====================================
        ============== RESULTS ==============
        =====================================
        Passes:      3
        Failures:    1
        Duration:    0s
        Total tests: 4
        
        FAIL
```